flake.nix

```diff
@@ -3,7 +3,6 @@
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-23.11";
```

Comment:
```suggestion
    nixpkgs.url = "github:auxolotl/nixpkgs/nixos-23.11";
```
I think we can change this :3
Then again we have not got much of a workflow around backporting. So might be better to hold off on this one.

Reply:
How about we switch it to unstable?

Reply:
That is certainly possible